src/services/lock/service/lock.service.ts

```diff
@@ -118,131 +102,35 @@ export class LockService {
-      // await this.lockRepository.deleteLock(id);
-
-      delete this.registeredLocks[id]; // Also remove it from the in-memory storage
-
+      delete this.registeredLocks[id];
       this.#logger.log(`Lock ${id} has been successfully unregistered.`);
     } else {
       this.#logger.log(`Lock ${id} is not registered to the current instance. Skipping.`);
     }
   }
 
-  async registerSelf() {
-    const dist1 = distance(Buffer.from('foo'), Buffer.from('bar'));
-    const nodes = await this.lockNodeRepository.distinct('node_id');
-    const nodesClosest = nodes
-      .map((e): [number, string] => {
-        let distance = 0;
-        let i = 0;
-        const min = Math.min(this.identity.id.length, e.node_id.length);
-        const max = Math.max(this.identity.id.length, e.node_id.length);
-        for (; i < min; ++i) {
-          distance = distance * 256 + ((this.identity.id as any)[i] ^ e[i]);
-        }
-        for (; i < max; ++i) distance = distance * 256 + 255;
-        return [distance, e.node_id];
-      })
-      .sort((a, b) => {
-        return a[0] - b[0];
-      })
-      .filter((e) => {
-        return e[1] !== this.identity.id;
-      });
-    const date = new Date();
-    const nodeReady = nodesClosest[date.getMinutes() % nodesClosest.length];
-    // console.log('nodeReady', nodeReady)
-    // if(nodeReady[1] === this.identity.id) {
-    //     console.log('Can register services')
-    // }
-  }
-
-  /**
-   * Attempts to reactivate services that are dead and activating fail over.
-   */
-  async faultCheck() {
-    // Logger.info('fault checking')
-    const runningServices = await this.lockRepository.fetchServicesLastPingedBefore10Minutes();
-    // console.log(runningServices)
-    for (const srv of runningServices) {
-      if (!srv.registered_id) continue;
-      // console.log(srv)
-      const destined_id = await this.nextNodeSelection(srv.registered_id);
-      // console.log(destined_id, this.identity.id)
-      if (destined_id === this.identity.id) {
-        //Can register services
-        if (this.regSrv[srv.id]) {
-          await this.registerLock(srv.id);
-          this.regSrv[srv.id].handler();
-          this.registeredLocks[srv.id] = {};
-        }
-      }
-    }
-  }
-
-  async registerOwnServices() {
-    for (const [id, val] of Object.entries(this.regSrv)) {
-      const currentLock = await this.lockRepository.findOneById(id);
-      if (!currentLock) {
-        await this.registerLock(id);
-        this.regSrv[id].handler();
-        this.registeredLocks[id] = {};
-      }
-    }
-  }
-
   async onModuleInit(): Promise<void> {
-    this._update_pid = setInterval(async () => {
-      for (const [id, lock] of Object.entries(this.registeredLocks)) {
-        await this.lockRepository.findOneByIdAndUpdateRegisteredIdOrIgnore({
-          id,
-          registered_id: this.identity.id,
-        });
-      }
-
-      await this.faultCheck();
-      await this.registerOwnServices();
-    }, 60 * 1000);
```

Comment:
What does this do? Why was it deleted? Please resolve if this was previously unused.